nodes/containers/nodes-containers-sysctls.adoc

```diff
@@ -17,10 +17,7 @@ Moreover, only those sysctls considered _safe_ are whitelisted by default; you
 can manually enable other _unsafe_ sysctls on the node to be available to the
 user.
 
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Node Tuning Operator]
+If you are setting the sysctl and it is not node-level, you can find information on how to do this in the section xref:../../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Using the Node Tuning Operator]
```

Comment:
- To avoid using "how to do this"

>If you are setting the sysctl and it is not node-level, you can find information in the xref:../../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Using the Node Tuning Operator] section on how to set..... 

- Alternatively

>If you are setting the sysctl and it is not node-level, you can find information on how to do this in the xref:../../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Using the Node Tuning Operator] section.

Reply:
Excellent suggestion! I've opted to eliminate "how to do this" completely!